src/Functions/hilbertDecode.h

```diff
@@ -0,0 +1,221 @@
+#pragma once
```

Comment:
Please consider writing this `.h` code into the corresponding `.cpp` file. The `hilbertDecode.cpp` has no code implementation in it, only a description, so it will be more optimal to put everything into the `.cpp` and remove the `.h` file at all, as it was done with almost all other functions in the `Functions` directory.

Reply:
I did this mostly for the unit test i wrote, so i can include this file. but if you insist, i'll include .cpp file there